modules/binary-tuple/src/test/java/org/apache/ignite/internal/binarytuple/BinaryTupleTest.java

```diff
@@ -74,6 +75,23 @@ public void nullValueTest() {
         assertNull(reader.timestampValue(0));
     }
 
+    /**
+     * Test boolean value encoding.
+     */
+    @Test
+    public void booleanTest() {
+        boolean[] values = {false, true};
+        for (boolean value : values) {
```

Comment:
nitpicking: it's better to use `@ParameterizedTest` for such cases (see `org.apache.ignite.internal.tx.TxManagerTest#testTestOnlyPendingCommit` for example)